waiter/config-shell.edn

```diff
@@ -1,4 +1,7 @@
 {
+ :metrics-consumer {
+                    :metrics-service-urls ["http://localhost:9001"]
```

Comment:
agreed. I left this in because I set up a dummy nodejs metrics service. going to remove this for now, and in the future add a mock service that we can test against